app/views/npq_separation/admin/applications/show.html.erb

```diff
@@ -1,286 +1,172 @@
-      row.with_value(text: @application.number_of_pupils.to_i)
-      row.with_action
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: 'Targeted support funding primary plus eligibility')
-      row.with_value{ boolean_red_green_tag(@application.tsf_primary_plus_eligibility) }
-      row.with_action
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Targeted delivery funding eligibility")
-      row.with_value { boolean_red_green_tag(@application.targeted_delivery_funding_eligibility) }
+      row.with_key(text: "Employment type")
+      row.with_value(text: @application.employment_type.try(:humanize).presence || "-")
     end
-
     sl.with_row do |row|
-      row.with_key(text: 'Funding choice')
-      row.with_value(text: @application.funding_choice&.capitalize || "-")
+      row.with_key(text: "ITT Lead mentor")
+      row.with_value(text: @application.lead_mentor ? "Yes" : "No")
     end
-
     sl.with_row do |row|
-      row.with_key(text: "Schedule Cohort")
-      if @application.cohort.present?
-        row.with_value(text: @application.cohort.start_year)
-        row.with_action(text: "Change", href: npq_separation_admin_applications_change_cohort_path(@application))
-      else
-        row.with_value(text: "-" )
-      end
+      row.with_key(text: "ITT provider")
+      row.with_value(text: @application.itt_provider.try(:operating_name).presence || "N/A")
     end
-
     sl.with_row do |row|
-      row.with_key(text: 'Schedule identifier')
-      row.with_value(text: @application.schedule.try(:identifier) || '-')
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Notes")
-      row.with_value do
-        @application.notes.present? ? simple_format(@application.notes) : "No notes"
-      end
+      row.with_key(text: "Country")
+      row.with_value(text: @application.teacher_catchment_country.presence || "-")
     end
   end
 %>
 
-<hr class="govuk-section-break govuk-section-break--l">
-
-<h2 class="govuk-heading-m">Declarations</h2>
-
-<% @declarations.each do |declaration| %>
-  <%= govuk_summary_card(title: declaration.declaration_type.humanize) do |summary_card|
-    concat(govuk_summary_list do |sl|
-
-      sl.with_row do |row|
-        row.with_key(text: "Declaration ID" )
-        row.with_value { declaration.ecf_id || "-" }
-      end
-
-      sl.with_row do |row|
-        row.with_key(text: "Declaration type")
-        row.with_value(text: declaration.declaration_type.humanize)
-      end
+<div data-declarations>
+  <h2 class="govuk-heading-l">Declarations</h2>
 
-      sl.with_row do |row|
-        row.with_key(text: "Declaration date")
-        row.with_value(text: declaration.declaration_date.to_fs(:govuk_short))
-      end
+  <% @declarations.each do |declaration| %>
+    <%- title = "#{declaration.declaration_type.humanize} (#{declaration.state.humanize})"%>
+    <%= govuk_summary_card(title:) do |summary_card|
+      concat(govuk_summary_list do |sl|
```

Comment:
Why does this need concat? you should be able to use `summary_card.with_summary_list`

https://govuk-components.netlify.app/components/summary-list/#summary-cards